### PR TITLE
Update kernel configs

### PR DESCRIPTION
**Pull request by raduiliescu**
## Reason for This PR

The current kernel config does no longer work with latest 4.14 LTS kernel. When compiling a kernel, compilation will stop asking for configuration parameters.

## Description of Changes

Regenerate configs using make menuconfig with kernel 4.14.174. For X86 add support for CTRL_ALT_DEL -  see https://github.com/firecracker-microvm/firecracker/blob/master/docs/api_requests/actions.md#sendctrlaltdel

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided (issue no. or explanation).
- [x] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.

### CODE REVIEW

File: resources/microvm-kernel-config [hunk 1]
```diff
@@ -1455,7 +1488,19 @@ CONFIG_VIRTIO_CONSOLE=y
 # CONFIG_SPI is not set
 # CONFIG_SPMI is not set
 # CONFIG_HSI is not set
-# CONFIG_PPS is not set
+CONFIG_PPS=y
```
Comment (iulianbarbu) [inline resources/microvm-kernel-config:1491]: I see that before it was not set. I guess regenerating the kernel config has set it on. I am wondering, besides drivers which are used by Firecracker, is there another criteria which come into play when adding drivers in the Firecracker example kernel config for amd64? Is the `CONFIG_PPS=y` worth to add?

Comment (raduiliescu) [reply resources/microvm-kernel-config:1491]: According to make menuconfig this option was automatically selected.
 

> Symbol: PPS [=y]                                                                                                                                                                                    
>    Type  : tristate                                                                                                                                                                                    
>    Prompt: PPS support           
>        .....                                                                                                                                                                       
>        Selected by: PTP_1588_CLOCK [=y] && NET [=y] && POSIX_TIMERS [=y]

Comment (iulianbarbu) [reply resources/microvm-kernel-config:1491]: > According to make menuconfig this option was automatically selected.
> 
> > Symbol: PPS [=y]
> > Type  : tristate
> > Prompt: PPS support
> > .....
> > Selected by: PTP_1588_CLOCK [=y] && NET [=y] && POSIX_TIMERS [=y]

I think we can let it through. The kernel defaults should do if it is not something flagrant.

**Comment by raduiliescu** [comment]
> 
> 
> LGTM! What do you say about also modifying the names of the config to something more suggestive and symetrical: `microvm-kernel-config-aarch64` and `microvm-kernel-config-x86_64`?

Is a good idea, just that I would do two modifications move config at the end after the dot like microvm-kernel-x86_64.config and I would rename the arm one microvm-kernel-arm64.config. arm64 being the name of the kernel architecture.

**Comment by dianpopa** [comment]
> > LGTM! What do you say about also modifying the names of the config to something more suggestive and symetrical: `microvm-kernel-config-aarch64` and `microvm-kernel-config-x86_64`?
> 
> Is a good idea, just that I would do two modifications move config at the end after the dot like microvm-kernel-x86_64.config and I would rename the arm one microvm-kernel-arm64.config. arm64 being the name of the kernel architecture.

Sounds good!

**Comment by raduiliescu** [comment]
> 
> 
> > > LGTM! What do you say about also modifying the names of the config to something more suggestive and symetrical: `microvm-kernel-config-aarch64` and `microvm-kernel-config-x86_64`?
> > 
> > 
> > Is a good idea, just that I would do two modifications move config at the end after the dot like microvm-kernel-x86_64.config and I would rename the arm one microvm-kernel-arm64.config. arm64 being the name of the kernel architecture.
> 
> Sounds good!

Done

**Pull request merged**